lib/matplotlib/patheffects.py

```diff
@@ -159,7 +159,26 @@ class Normal(AbstractPathEffect):
     The Normal PathEffect's sole purpose is to draw the original artist with
     no special path effect.
     """
-    pass
+
+
+def _subclass_with_normal(effect_class):
+    """
+    Create a PathEffect class combining the effect of *cls* and a normal draw.
+    """
+
+    class withEffect(effect_class):
+        def draw_path(self, renderer, gc, tpath, affine, rgbFace):
+            super().draw_path(renderer, gc, tpath, affine, rgbFace)
+            renderer.draw_path(gc, tpath, affine, rgbFace)
+
+    withEffect.__name__ = f"with{effect_class.__name__}"
+    withEffect.__doc__ = f"""
+    Adds a `.{effect_class.__name__}` and then draws the original Artist to
```

Comment:
I've been struggling to understand what the `with...` classes are and what they are good for (path effect newbie here). Therefore I'm a bit picky on docs. I propose something like this (Please check ReST syntax and line length. This is a bit difficult in the github interface):

~~~
"""
A shortcut PathEffect for applying `.{effect_class.__name__}` and then
drawing the original Artist.

Examples
--------
With this class you can use ::

    artist.set_path_effects([path_effects.with{effect_class.__name__}()])

as a shortcut for ::

    artist.set_path_effects([path_effects.{effect_class.__name__}(),
                             path_effects.Normal()])
"""
~~~

Reply:
Taking for example the path effect proposed at #15458, using just `path_effects=[TickedStroke(...)]` would *only* draw the ticks, not the "original" path; using `path_effects=[withTickedStroke(...)]` draws both the original path and not the Stroke.  Your suggestion is fine, adding it.